tests/test-synopsis.php

```diff
@@ -161,12 +161,21 @@ public function testRender() {
 				'description' => 'If you are hungry between meals, you should snack.',
 				'optional'    => true,
 			),
+			array(
```

Comment:
use short array syntax

Reply:
I tried to match the syntax used for the rest of the arrays in the test case. Should I change the rest to short array syntax as well?

Reply:
looking at that file in general you introduced first short array here https://github.com/wp-cli/wp-cli/blob/13954af56bdbf4558fb6f310b0e359cf4c434128/tests/test-synopsis.php#L169

Reply:
My mistake - I was working off an outdated fork